deeplearning-models/modelcard-schema.json

```diff
@@ -0,0 +1,49 @@
+{
+   "type": "object",
+   "properties": {
+      "name": {"type": "string" },
+      "author": {"type": "string"},
+      "long_description": {
+         "type": "string",
+         "maxLength": 280
+      },
+      "short_description": {
+         "type": "string",
```

Comment:
Is this short description the source of the strings appended to track names?

Do we have any control over this schema?  Can we somehow provide optional translations of user-visible short and long descriptions into other locales?

Reply:
this is the short, 50-something character description shown on the scrolled window on the left. It helps users browse through models and pick the one suited for their needs. 

Because these models (along with their json metadata) are meant to be contributed by the community (and hosted in HuggingFace repositories), I can't think of a way to provide optional translations. Maybe querying a translation API?

<img width="1116" alt="Screen Shot 2022-01-11 at 12 38 22 PM" src="https://user-images.githubusercontent.com/55194054/149001811-e65c8c2d-fe0f-4b98-9f87-427035459c22.png">